pipeline/beam_tables.py

```diff
@@ -568,9 +581,15 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
       lines = _read_scan_text(p, new_filenames)
 
       if scan_type == satellite.SCAN_TYPE_SATELLITE:
+        received_tagging = True
+        # For Satellite v1 - v2.1, the received IP tags (e.g. asnum) are in a
+        # separate file from results.json. The steps for adding these tags are
```

Comment:
I don't think it's a good idea to have seperate behavior for dev and prod here. We're just going to end up pushing unnoticed bugs what didn't show up in dev.

If we think these tags aren't useful enough to be worth the cost and expense in prod then we shouldn't add them in dev either. It sounds like the actual relevant distinction is that we want to avoid adding them for v2.2 data, but that code should all be able to live in satellite.py

Since we have to decide the structure of the pipeline ahead of runtime this may require doing something like adding a partitioning step into pre2.2 rows and post-2.2 rows, then only running the tagging steps on the pre-2.2 rows. That means when we run nightly jobs over only post-2.2 data the tagging steps will appear in the pipeline structure, but 0 rows will ever flow through it.